mscore/plugin/api/part.h

```diff
@@ -25,10 +25,42 @@ namespace PluginAPI {
 
 class Part : public Ms::PluginAPI::ScoreElement {
       Q_OBJECT
-      Q_PROPERTY(int                            startTrack          READ startTrack)
-      Q_PROPERTY(int                            endTrack            READ endTrack)
-      /// The string identifier for the current instrument. \since MuseScore 3.2
-      Q_PROPERTY(QString                        instrumentId        READ instrumentId)
+      Q_PROPERTY(int                            startTrack           READ startTrack)
+      Q_PROPERTY(int                            endTrack             READ endTrack)
+      /// The string identifier for the current instrument. \since MuseScore 3.2.1
```

Comment:
`instrumentId` is already a part of 3.2 release, there is no need to change version number for it

Reply:
Dang it! I got carried away and stomped on my own prior commit. :)
Fixing it.

Reply:
Ok... pushed the change and removed the updated version to the plugin doxygen configuration file for now. It appears someone updated that in master already to 3.2. I don't want to break that.

-Dale

Reply:
The merge was an overlooked mistake on my part. It's gone now.
-Dale